README.md

```diff
@@ -64,7 +64,7 @@ abstract class Controller extends BaseController
     public function __construct()
     {
         # Default title
-        Meta::title('This is default page title to complete section title');
+        Meta::set('title', 'This is default page title to complete section title');
```

Comment:
`Meta::title` is a valid method and works as default tag to title, to complete custom section titles.

For example:

Meta::title('My Wonderfull Website');
Meta::set('title', 'Products');

Will print: `Products - My Wonderfull Website`

Reply:
If you need a default title, you can set only the `Meta::set('title'` on `BaseController` that will be overwritten on each custom endpoint :)